src/styles/blocks/card.scss

```diff
@@ -0,0 +1,110 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+
+  border: 1px solid $color-border;
+
+  padding-top: 32px;
+
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+
+  &__image {
+    padding-left: 19px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__info {
+    margin-left: 16px;
+  }
+
+  &__title {
+    margin: 40px 0 4px;
+    color: $color-text-black;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__code {
+    margin: 0;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $color-text-gray;
+  }
+
+  &__group1 {
+    margin-top: 16px;
+
+    @include centred-flexbox;
+  }
+
+  &__group1-reviews {
+    margin: 0 16px;
+    font-size: 10px;
+    font-weight: 400;
```

Comment:
This is default value so you can remove it

Reply:
When I deleted  the properties  margin: 0 16px and font-size: 10px -  the image shifts and the font increases. I deleted only  font-weight: 400;